term/fhir-term-graph-loader/src/main/java/com/ibm/fhir/term/graph/loader/util/ConfigLoader.java

```diff
@@ -16,16 +16,18 @@
 import org.apache.commons.configuration2.builder.fluent.Parameters;
 import org.apache.commons.configuration2.ex.ConfigurationException;
 
-/*
+/**
  * This class will load a Configuration from a property file, then override parameters based on environment variables.
  */
 public class ConfigLoader {
+    public static final String DATACENTER = "storage.cql.local-datacenter";
     public static final String STORAGE_HOSTNAME = "storage.hostname";
     public static final String STORAGE_PORT = "storage.port";
     public static final String STORAGE_USERNAME = "storage.username";
     public static final String STORAGE_PASSWORD = "storage.password";
     public static final String INDEX_SEARCH_HOSTNAME = "index.search.hostname";
     public static final String INDEX_SEARCH_PORT = "index.search.port";
+    public static final String DATACENTER_ENV = "TERM_STORAGE_DATACENTER";
```

Comment:
Maybe "STORAGE_DATACENTER_ENV" to follow the pattern?

Reply:
done